sdk/communication/communication-chat/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 1.0.2 (Unreleased)
```

Comment:
I am not sure about the version number. Would like some suggestions here. @xirzec @DominikMe  @juancamilor

Reply:
The swagger is now a `-preview` version, so should this have a beta suffix like in `1.1.0-beta.1`?

Leaving it up to Jeff about the minor bump. I tend to follow semver and would bump it, but ideally we could have more changes. How far along is enabling real-time notifications for browser?

Reply:
I have no strong feelings one way or the other. I'm totally fine with following semver to the letter if that works for you.

Reply:
My main concern is that we have a GA version referencing a beta swagger
`tag: package-chat-2021-04-05-preview6`

I think we have to fix that? 
@juancamilor @LuChen-Microsoft

Reply:
I used 1.1.0-beta.1 instead